src/UbuntuToolkit/sortfiltermodel.cpp

```diff
@@ -36,58 +36,65 @@ UT_NAMESPACE_BEGIN
  *
  * Example usage:
  * \qml
- * import QtQuick 2.4
- * import Ubuntu.Components 1.2
- * import Ubuntu.Components.ListItems 1.1
+ * import QtQuick 2.9
+ * import Ubuntu.Components 1.3
  *
  * MainView {
  *     width: units.gu(80)
  *     height: units.gu(40)
  *
  *     ListModel {
- *         id: movies
- *         ListElement {
- *             title: "Esign"
- *             producer: "Chris Larkee"
- *         }
- *         ListElement {
- *             title: "Elephants Dream"
- *             // lowercase b
- *             producer: "blender"
- *         }
- *         ListElement {
- *             title: "Big Buck Bunny"
- *             producer: "Blender"
- *         }
- *     }
- *
- *     SortFilterModel {
- *         id: sortedMovies
- *         model: movies
- *         sort.property: "title"
- *         sort.order: Qt.DescendingOrder
- *         // case insensitive sorting
- *         sortCaseSensitivity: Qt.CaseInsensitive
- *
- *         filter.property: "producer"
- *         // case insensitive matches
- *         filter.pattern: /blender/i
- *     }
- *
- *     ListView {
- *         model: sortedMovies
- *         anchors.fill: parent
- *
- *         delegate: Subtitled {
- *             text: title
- *             subText: producer
- *         }
- *
- *         section.delegate: ListItem.Header { text: i18n.tr(section) }
- *         section.property: "title"
- *         section.criteria: ViewSection.FirstCharacter
- *     }
- * }
+ *        id: movies
```

Comment:
The indentation is wrong, please double check it (and the diff will become much smaller).

Reply:
Thanks. Modified
Tell me if that's ok now